pytorch_lightning/callbacks/pruning.py

```diff
@@ -371,28 +383,29 @@ def on_before_accelerator_backend_setup(self, trainer, pl_module: LightningModul
             for i, (module, name) in enumerate(self._parameters_to_prune):
                 id_ = id(module)
                 self._original_layers.setdefault(id_, {"data": deepcopy(module), "names": []})
-                self._original_layers[id_]["names"].append((i, name))
+                self._original_layers[id_]["names"].append((i, name))  # type: ignore
```

Comment:
it says 'Tensor' not callable which I don't understand at all. I think it might be, because the type of the object we append something to might be inferred from torch.nn.Module as `Union[Tensor, Module]`